src/style.scss

```diff
@@ -1,21 +1,18 @@
-.wc-free-shipping-progress-bar {
-	#progressBar {
-		align-items: center;
-		background: transparent;
-		border: 1px solid #4c566a;
-		border-radius: 0.25em;
-		display: flex;
-		height: 1em;
-		justify-content: center;
-		margin-top: 0.5em;
-		padding: 0.1em;
-		position: relative;
-
-		#progress {
-			background: #4c566a;
-			height: 100%;
-			left: 0;
-			position: absolute;
-		}
-	}
+.progress-container {
+	align-items: center;
+	background: transparent;
+	border: 1px solid currentColor;
+	border-radius: 0.125em;
+	display: flex;
+	height: 2em;
+	justify-content: center;
+	margin-top: 0.5em;
+	position: relative;
 }
+
+.progress-bar {
+	background: #96588a;
```

Comment:
You're still providing a default color here, when you shouldn't.

Reply:
I thought that's what I need to do to get the default background color to work when someone clicks the `clear` button.  When removing this color, the progress bar is not visible when being added to the page and it becomes invisible when the `clear` button gets clicked. 

<img width="880" alt="Screenshot 2021-09-30 at 23 12 27" src="https://user-images.githubusercontent.com/3323310/135492273-0f755f44-75e4-4cf0-bcfc-ecfd984a41cf.png">

How can I remove the color code while ensuring that the progress bar has a default color when getting added and when the `clear` button gets clicked?

Reply:
How about using `currentColor` there, and it would get the text color?

Reply:
> How about using currentColor there, and it would get the text color?
Thanks for pointing this out. I haven't thought about using `currentColor` in such a way. I just adjusted it.